x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_router.tsx

```diff
@@ -14,13 +14,14 @@ import { getEngineBreadcrumbs } from '../engine';
 
 import { CRAWLER_TITLE } from './constants';
 import { CrawlerLanding } from './crawler_landing';
+import { CrawlerOverview } from './crawler_overview';
 
 export const CrawlerRouter: React.FC = () => {
   return (
     <Switch>
       <Route>
         <SetPageChrome trail={getEngineBreadcrumbs([CRAWLER_TITLE])} />
-        <CrawlerLanding />
+        {process.env.NODE_ENV === 'development' ? <CrawlerOverview /> : <CrawlerLanding />}
```

Comment:
I think maybe a feature flag separate from `NODE_ENV` maybe better here.

I think by default we should get the same thing *in development* that we are going to get *in production*. So make opting into CrawlerLanding an explicit choice, rather than the default in development, to avoid confusion when testing.

just moving it to something like `process.env.ENABLE_APP_SEARCH_CRAWLER` (or something similar) *should* work.

Reply:
This is a temporary measure until this feature is finished, We will not ship the landing page with v7.15

Reply:
Yep, I realize it's temporary. I just think it's better to move this to a specific feature flag, and not use NODE_ENV. Like if I'm running the code now I'd rather have the actual code that will actually be running in 7.14 locally, not 7.15 code.

I'm not going to block your PR on this if you feel strongly about this...

Reply:
Do you see what I'm saying? Like if you're a developer and you don't know about this, you'd just assume the code you're running is the code that will be in the next release, when in fact it's not.